SwiftWisdom/Rx/Rx+Extensions.swift

```diff
@@ -47,6 +47,21 @@ public func <- <T>(observer: Variable<T>, observable: Variable<T>) -> Disposable
     return observer <- observable.asObservable()
 }
 
+public func <- <T: ObserverType, O>(observer: T, behaviorRelay: BehaviorRelay<O>) -> Disposable where T.E == O {
+    return observer <- behaviorRelay.asObservable()
+}
+
+public func <- <T: ObserverType, O>(observer: T, behaviorRelay: BehaviorRelay<O>) -> Disposable where T.E == O? {
+    return observer <- behaviorRelay.asObservable()
+}
+
+public func <- <T, O: ObservableType>(behaviorRelay: BehaviorRelay<T>, observable: O) -> Disposable where O.E == T {
+    return observable.bind(to: behaviorRelay)
+}
+
+public func <- <T>(observer: BehaviorRelay<T>, observable: BehaviorRelay<T>) -> Disposable {
+    return observer <- observable.asObservable()
+}
```

Comment:
Add a new line after this func to maintain the existing pattern of a new line between each func.